fe2-android/app/src/main/res/values/strings.xml

```diff
@@ -310,6 +310,8 @@
   <string name="error_post_transaction">Could not post a transaction\n%1$s</string>
   <string name="error_no_rollcall_closed_in_LAO">Error no RollCall closed in the LAO\n%1$s</string>
   <string name="error_public_key_receiver">Error on the receiver s public key\n%1$s</string>
+  <string name="qr_code_not_pop_token">Error, this QR code is not your pop token</string>
```

Comment:
Maybe "not **a** (valid) pop token"?

Reply:
We can't diffentiate there for a valid or unvalid pop token, only a pop token or not one

Reply:
that is what i meant but yes, you are right. the word "valid" is not appropriate here 😅